graph/export.go

```diff
@@ -61,6 +62,21 @@ func (s *TagStore) ImageExport(names []string, outStream io.Writer) error {
 				// and will not need to be added to this map
 				if len(repoTag) > 0 {
 					addKey(repoName, repoTag, img.ID)
+				} else {
+					//This must be an ID. Here repoName is an ID.
+					nonTruncID, err := s.graph.GetTruncIndex().Get(repoName)
+					if err != nil {
+						return err
+					}
+					reverseLookupMap := s.ByID()
+					for id, repos := range reverseLookupMap {
+						if id == nonTruncID {
+							for _, repo := range repos {
+								index := strings.Index(repo, ":")
```

Comment:
oh, I'm very sorry, I should have thought about this before, isn't this like doing?

```go
repoName, repoTag := strings.SplitN(repo, ":", 2)
addKey(repoName, repoTag, id)
```